src/commands/dev/client.ts

```diff
@@ -120,3 +127,24 @@ async function buildWebpackServer(config: NormalizedServiceConfig) {
 
     return server;
 }
+
+function subscribeToManifestReadyEvent(compiler: webpack.Compiler, onManifestReady: () => void) {
```

Comment:
Why we don't just wait for webpack with all child compilations to finish?

Reply:
Because we don't want to wait for, for example, ts checker or statosope.